src/Factory.ts

```diff
@@ -7,8 +8,11 @@ import {ExpressServer} from "./server/ExpressServer";
  * @param expressApp Express application instance
  * @param requireDirs Array of directories where from controller files will be loaded
  */
-export function registerActionsInExpressApp(expressApp: any, requireDirs?: string[]): ControllerRunner {
+export function registerActionsInExpressApp(expressApp: any, requireDirs?: string[], container?: Container): ControllerRunner {
```

Comment:
I can understand your concern here. Alternatively, there could be a separate method that provides the framework with a global container. This way you would not have to alter this method's API. What do you think?